shiny/input_handler.py

```diff
@@ -52,18 +52,6 @@ def _process_value(
 for more information on how to create custom input bindings. (The article is about
 Shiny for R, but the JavaScript and general principles are the same.)
 
-Methods
```

Comment:
I think I deleted this while pairing w/ Barret, but can put back in. I think it caused griffe to error when parsing the docstring (but could be wrong).